tests/robot-tests/tests/admin/bau/prerelease.robot

```diff
@@ -155,18 +155,79 @@ Go to prerelease access page
     user goes to url    ${RELEASE_URL}/prerelease-access
     user waits until h2 is visible    Manage pre-release user access
 
-Invite users to prerelease for scheduled release
-    # This is GOV.UK Notify's test email address
-    user enters text into element    css:input[name="email"]    simulate-delivered@notifications.service.gov.uk
-    user clicks button    Invite new user
-
+Validate the invite emails field is required
+    user clicks button    Invite new users
+    user waits until element contains    id:preReleaseUserAccessForm-emails-error
+    ...    Please enter 1 or more email addresses
+
+Validate the invite emails field only accepts @education.gov.uk email addresses
+    ${emails}=    Catenate    SEPARATOR=\n
+    ...    ees-analyst1@education.gov.uk
+    ...    test@test.com
+    user enters text into element    css:textarea[name="emails"]    ${emails}
+    user clicks button    Invite new users
+    user waits until element contains    id:preReleaseUserAccessForm-emails-error
+    ...    Please enter valid @education.gov.uk email addresses
+
+Invite users to the prerelease
+    ${emails}=    Catenate    SEPARATOR=\n
+    ...    simulate-delivered@notifications.service.gov.uk
+    ...    ees-analyst1@education.gov.uk
+    user enters text into element    css:textarea[name="emails"]    ${emails}
+    user clicks button    Invite new users
+    ${modal}=    user waits until modal is visible    Confirm pre-release invitations
+    user waits until element contains    ${modal}    Email notifications will be sent immediately
+
+    user waits until page contains element    css:[data-testid="invitableList"] li    limit=2
```

Comment:
Could use the `user checks list has x items` keyword for this (and other similar instances):

```robot
user checks list has x items  testid:invitableList  2
```

Note that we're using our custom `testid` selector, which is less verbose than the equivalent `css` or `xpath` selectors

Reply:
Cool, I didn't know about our `testid` selector, I've made this change. Thanks 🙂